logstash-core-event-java/build.gradle

```diff
@@ -1,104 +1,108 @@
 buildscript {
-  repositories {
-    mavenLocal()
-    mavenCentral()
-    jcenter()
-  }
-  dependencies {
-    classpath 'net.saliman:gradle-cobertura-plugin:2.2.8'
-    classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.2'
-  }
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        jcenter()
+    }
+    dependencies {
+        classpath 'net.saliman:gradle-cobertura-plugin:2.2.8'
+        classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.2'
```

Comment:
if you mean the jar we produce (org.logstash namespace) no, I don't see any good reason to shadow